distribution/lib/Standard/Base/0.0.0-dev/src/Data.enso

```diff
@@ -184,15 +188,10 @@ list_directory directory:File (name_filter:(Text | Nothing)=Nothing) recursive:B
          file = enso_project.data / "spreadsheet.xls"
          Data.fetch URL . body . write file
 @uri Text_Input
-fetch : (URI | Text) -> HTTP_Method -> Vector (Header | Pair Text Text) -> Boolean -> Any ! Request_Error | HTTP_Error
-fetch (uri:(URI | Text)) (method:HTTP_Method=HTTP_Method.Get) (headers:(Vector (Header | Pair Text Text))=[]) (try_auto_parse_response:Boolean=True) =
-    response = HTTP.fetch uri method headers
-    if try_auto_parse_response.not then response.with_materialized_body else
-        ## We cannot catch decoding errors here and fall-back to the raw response
-           body, because as soon as decoding is started, at least part of the
-           input stream may already be consumed, so we cannot easily reconstruct
-           the whole stream.
-        response.decode if_unsupported=response.with_materialized_body
+@format Data_Read_Helpers.format_widget_with_raw_response
+fetch : (URI | Text) -> HTTP_Method -> Vector (Header | Pair Text Text) -> File_Format -> Any ! Request_Error | HTTP_Error
+fetch (uri:(URI | Text)) (method:HTTP_Method=HTTP_Method.Get) (headers:(Vector (Header | Pair Text Text))=[]) (format = Auto_Detect) =
+    Data_Read_Helpers.fetch_following_data_links uri method headers format
```

Comment:
As discussed can we map if `format=False` to `Raw_Request` or `format=True` to `Auto_Detect`.
And attach a deprecated warning please.

Reply:
Done.

I did not add tests since it's just legacy, but I tested manually:
![image](https://github.com/enso-org/enso/assets/1436948/465b5a55-49f1-49d0-8aa4-1573ce609d95)
![image](https://github.com/enso-org/enso/assets/1436948/d5235b20-ed89-46d3-a7a4-581d7f1062d1)